docs/trino-connector/development.md

```diff
@@ -118,6 +139,29 @@ then you can see the `gravitino-trino-connecor` source files and directories in
             <artifactId>jakarta.validation-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr4-runtime</artifactId>
+            <version>4.9.2</version>
+        </dependency>
```

Comment:
Why do we need to add these dependencies?

Reply:
The dependency is not used here, I'll remove it.